src/core/layout/footer/style.js

```diff
@@ -1,14 +1,14 @@
 import { makeStyles } from '@material-ui/core'
 
 const position = theme => ({
-  position: 'absolute',
-  width: '100vw',
   bottom: theme.spacing(0),
+  position: 'absolute',
+  width: '100%',
 })
 
 export const useStyles = makeStyles(theme => ({
-  title: { color: 'white' },
-  footer: {
+  content: { color: 'white' },
```

Comment:
can you extract this in a separate style ? under content folder?

Reply:
Done